tests/plugins/imagebase/features/link.js

```diff
@@ -410,6 +410,33 @@
 				}
 			} );
 		},
+
+		// #tp3298
+		'test unlink with data callback after it': function( editor, bot ) {
```

Comment:
I'm not convinced if this name conveys test purpose here.

It's testing whether calling setData with any (potentially irrelevant) data won't modify `data.link`.

`test calling setData after unlink keeps correct data.link`?